test-common/build.gradle

```diff
@@ -3,4 +3,5 @@ description = 'Test utility library, generic test utilities useful for TripleA p
 dependencies {
     implementation "org.hamcrest:java-hamcrest:$hamcrestVersion"
     implementation "org.junit.jupiter:junit-jupiter-api:$junitJupiterVersion"
+    implementation "org.apache.commons:commons-lang3:$apacheCommonsLangVersion"
```

Comment:
I don't think this dependency is used in this PR

Reply:
Thanks - removed in: 7bb93e7ba